### PR TITLE
Add form security token to contact actions

### LINKED ISSUE #10672: Add security token check to /contact module
Discovered while working on #10502

### PR DESCRIPTION
**Pull request by MrPetovan**
Closes #10672 
- Add token to batch POST actions
- Add token to individual GET actions

### CODE REVIEW

**Review by tobiasd — APPROVED** [review]

**Comment by MrPetovan** [comment]
Pinging @annando

**Comment by annando** [comment]
@MrPetovan sorry, I had been busy.

File: src/Module/Contact.php [hunk 1]
```diff
@@ -840,6 +846,7 @@ public static function content(array $parameters = [], $update = 0)
 			'$submit'     => DI::l10n()->t('Find'),
 			'$cmd'        => DI::args()->getCommand(),
 			'$contacts'   => $contacts,
```
Comment (annando) [inline src/Module/Contact.php:848]: Shouldn't all value be indented to the same level?

**Review by annando — COMMENTED** [review]
Just some code style question ...

File: src/Module/Contact.php (hunk 1 above)
Comment (MrPetovan) [reply src/Module/Contact.php:848]: It's debatable when the key length varies a lot, like in this case. Evidently a choice had been made earlier to align short keys together and leave longer keys alone and I chose to respect it.

**Comment by tobiasd** [comment]
Does blocking / ignoring authors from the network stream (I'm using vier, frio not checked) need some additional work? When I follow the link from the pull-down menu I get an error message about a too old security token.

**Comment by annando** [comment]
@tobiasd I guess best is to create an issue to not forget about it. This is surely something that needs to be fixed before the release.

**Comment by tobiasd** [comment]
> @tobiasd I guess best is to create an issue to not forget about it. This is surely something that needs to be fixed before the release.

done

**Pull request merged**